roles/confluent.kafka_broker/tasks/restart_kafka.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Populate Service Facts
+  service_facts:
+
+- block:
```

Comment:
Can you add a comment block that explains how this code works, it's quite counter intuitive to how I think someone would approach this.

Reply:
the docs to me are pretty straight forward (and also easy to find, they are the first result when searching "ansible blocks")
https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html

we dont currently put comments on how modules work, where should we draw the line?

Reply:
We should be commenting on complex/not easy to understand pieces of code.  In this instance, it's the "Restart confluent-server Serially" I am referring to.  Not sure my previous comment was clear enough in that regard, apologies.

Reply:
how does it look now?